release_notes_generator/model/record/issue_record.py

```
@@ -52,10 +52,39 @@ def is_open(self) -> bool:
+
+        # Assignees (main implementers)
+        for assignee in self.assignees:
+            devs.add(f"{assignee}")
+
+        # Linked PR authors (people who created PRs closing this issue)
+        for pr in self._pull_requests.values():
+            author = getattr(pr.user, "login", None)
+            if author:
+                devs.add(f"@{author}")
+
+            for cid, commit in self._commits[pr.number].items():
```

Ruff (0.13.3)

Loop control variable `cid` not used within loop body

Rename unused `cid` to `_cid`

(B007)